docs/source/GetStartedGuide.rst

```diff
@@ -140,9 +140,6 @@ The output might look like this::
 These Python files are considered by Avocado to contain ``INSTRUMENTED``
 tests.
 
-.. These should refer to proper simple tests example but they are currently
-   lacking in our tree. See GitHub issue #628.
-
```

Comment:
Yes, I'm sorry, you're right and I need to try rpm-version more often... And while looking at it, let me add new issue: https://trello.com/c/VFcxxwIp/656-bug-install-simple-tests-data-files-in-the-correct-location